translation.rst

```diff
@@ -467,6 +467,26 @@ The ``translation:extract`` command looks for missing translations in:
 
         $ composer require nikic/php-parser
 
+By default, the ``translation:extract`` command extracts new messages in
+*message/translation* pairs, with the translation populated with the same
+text as the message, prefixed by ``__``. You can customize this prefix using
+the ``--prefix`` option.
+
+If you want to leave new translations empty, use the ``--no-fill`` option.
+When enabled, only messages are created, and the translation strings are left
+empty. This is particularly useful when using external translation tools, as
+it ensures untranslated strings are easily identifiable without any pre-filled
+content. Note that when ``--no-fill`` is used, the ``--prefix`` option has
+no effect.
+
+.. code-block:: terminal
+
+    # change default prefix for new translations
```

Comment:
```suggestion
    # changes default prefix for new translations
```

Reply:
Done, I also shorten the comment to "changes default prefix"